panel/src/components/Views/SystemView.vue

```diff
@@ -106,7 +103,18 @@ export default {
 
 		await promiseAll(promises);
 
-		console.info("System health checks ended.");
+		console.info(
+			`System health checks ended. ${
+				promises.length - this.accessible.length
+			} issues found.`
+		);
+
+		// print exceptions from the update check to console for debugging
+		if (this.exceptions.length > 0) {
+			console.info("----");
+			console.info("Running plugin version checks for the Panel system view:");
```

Comment:
Not just plugins, could also be Kirby itself:

```suggestion
			console.info("Running the update check for Kirby and plugins.");
```